dev-client/src/translations/en.json

```diff
@@ -652,7 +653,7 @@
     }
   },
   "image": {
-    "use_camera": "Use Camera",
-    "choose_from_gallery": "Choose From Gallery"
+    "use_camera": "Take Photo",
+    "choose_from_gallery": "Photo Library"
```

Comment:
@CourtneyLee333 do you have thoughts on changing this copy?

Reply:
I'm seeing a lot of different terminology across platforms and apps. I'm fine to start with the suggested text shown here, and see how it works out.